descqa/shear_test.py

```diff
@@ -273,7 +273,7 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
 
         pars.set_cosmology(H0=cosmo.H0.value, ombh2=cosmo.Ob0*cosmo.h**2, omch2=(cosmo.Om0-cosmo.Ob0)*cosmo.h**2)
         pars.InitPower.set_params(ns=ns, As=2.168e-9*(s8/0.8)**2)
-        camb.set_halofit_version(version='takahashi') # pylint: disable=no-member
+        #camb.set_halofit_version(version='takahashi') # pylint: disable=no-member
```

Comment:
Is this line no longer needed? If so, remove it instead of commenting it out (because the residual comment makes people wonder why this is commented out and may cause future confusion. 
```suggestion
```

Reply:
@patricialarsen suggested this when I was testing for another problem. I defer to her on whether this line should be reinstated or removed.

Reply:
So the change is from an update to camb. IIRC the syntax is appropriate for the older version but will cause a crash in the new version. 

Let me double check what the default nonlinear model is for the matter power interpolator in case we need to replace this with the updated syntax.